### PR TITLE
wms_integrations

### PR DESCRIPTION
**Pull request by zamb124**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by yelizariev** [comment]
Consider to send your modules to some oca repository, e.g. to https://github.com/OCA/l10n-russia

**Comment by smetl** [comment]
@jco-odoo

**Review by jco-odoo — COMMENTED** [review]
@zamb124 For the modules with openapi and wms_integrations, that would indeed be more OCA material.  For l10n_ru(_doc), it would of course be great to have a Russian localization, but at first sight, it looks like a lot could be done in a simpler way certainly when it is about templates of documents.  I don't know if you are open to discussing further about what is needed for Russia.

File: addons/l10n_ru/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    'description': """
+This is the base module to manage the accounting chart for Russia in OpenERP.
+==============================================================================
+Возможности:
+
+  - План счетов бухгалтерского учёта финансово-хозяйственной деятельности организаций, утверждённый Приказом Минфина РФ от 31.10.2000 года № 94н
+    """,
+    'author': 'CodUP',
+    'website': 'http://codup.com',
+    'license': 'AGPL-3',
```
Comment (yelizariev) [inline addons/l10n_ru/__manifest__.py:23]: I'm not sure that Odoo s.a. is ready to take this module with restrictions of AGPL licence. 
Also, these module is published in apps store:

https://apps.odoo.com/apps/modules/13.0/l10n_ru/
https://apps.odoo.com/apps/modules/13.0/l10n_ru_doc/

I suppose we need authors authorization to proceed

@wowas18 ?

**Comment by C3POdoo** [comment]
Dear @zamb124,

Thank you for your contribution but the version 14.0 is no longer supported.
We only support the last 3 stable versions so no longer accepts patches into this branch.

We apology if we could not look at your request in time.
If the contribution still makes sense for the upper version, please let us know and do not hesitate to recreate one for the recent versions. We will try to check it as soon as possible.

_This is an automated message._

**Pull request closed without merging**